src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -666,15 +666,121 @@ The CSS approach, by keeping specificity low on base class properties and resett
   {{/l-flex-item}}
 {{/l-flex}}
 ```
+
+### Ordering
+
+Ordering - Ordering can be applied to nested <code>.pf-l-flex</code> and <code>.pf-l-flex__item</code>s. Spacing may need to be managed based on how items are ordered. Because order could apply to an innumerable number of elements, order is set inline as `--pf-l-flex--item--Order{-on-[breakpoint]}: {int}`.
```

Comment:
to align with the doc table

```suggestion
Ordering - Ordering can be applied to nested <code>.pf-l-flex</code> and <code>.pf-l-flex__item</code>s. Spacing may need to be managed based on how items are ordered. Because order could apply to an innumerable number of elements, order is set inline as `--pf-l-flex--item--Order{-on-[breakpoint]}: {order}`.
```